docs/website/root/mithril/advanced/mithril-protocol/security.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 3
-sidebar_label: Security
+sidebar_label: Protocol security
```

Comment:
In this section, I'd also suggest dropping some definitions to avoid repetition.
For example:

- Delete the intro paragraph (Mithril is a stake-based threshold multi-signature (STM) protocol that aggregates individual signatures into a compact certificate. This process occurs when the total stake supporting a message exceeds a predefined threshold. The protocol enhances scalability in signing, communication, and verification by pseudorandomly selecting a subset of eligible participants to sign each message.)
- Start with 'This document...'
- Consider linking out to 'Protocol phases' instead of summarizing them in 'Mithril protocol explained' and 'Protocol phases'
- Continue with the 'Adversarial model' and everything below

Reply:
Done 👍